src/components/Config/Config.tsx

```diff
@@ -79,120 +78,67 @@ function Config() {
-      const currentChainId = await window.ethereum.request({ method: `eth_chainId` });
-      if (currentChainId === arbitrumOneChainId) {
+  const handleConnect = useCallback(async () => {
+    const isMetamaskInstalled = typeof window.ethereum !== 'undefined';
+    if (isMetamaskInstalled) {
+      if (network === Network.Ethereum) {
+        setSubmitted(true);
+        setCompleted(false);
+      } else if (network === Network.Arbitrum) {
+        setSubmitted(true);
+        setCompleted(false);
         setGasPriceAware(false);
         setDraft(true);
         setConfirmationTime(0);
-        setNetwork(Network.Arbitrum);
-        notification.success({ message: `You're connected to Automate!` });
-      } else {
-        try {
-          await window.ethereum.request({
-            method: 'wallet_addEthereumChain',
-            params: [
-              {
-                chainId: arbitrumOneChainId,
-                chainName: 'Arbitrum One',
-                rpcUrls: ['https://arb1.arbitrum.io/rpc'],
-                blockExplorerUrls: ['https://arbiscan.io/'],
-                nativeCurrency: {
-                  symbol: 'ETH', // 2-6 characters long
-                  decimals: 18,
-                },
-              },
-            ],
-          });
-        } catch (addError) {
-          console.log(addError);
-        }
       }
     } else {
-      notification.error({
-        message: (
-          <span>
-            Metamask is not installed. Metamask is required to connect Automate. Install the{' '}
-            <a href="https://metamask.io/download/" target="_blank" rel="noopener noreferrer">
-              Metamask extension.
-            </a>{' '}
-            If you have installed refresh the page.
-          </span>
-        ),
-      });
-    }
-  }, []);
-
-  const handleArbitrumConnection = useCallback(async () => {
-    const isMetamaskInstalled = typeof window.ethereum !== 'undefined';
-    if (isMetamaskInstalled) {
-      arbitrumNetworkConnect();
-    } else {
-      notification.error({
-        message: (
-          <span>
-            Metamask is not installed. Metamask is required to connect Automate. Install the{' '}
-            <a href="https://metamask.io/download/" target="_blank" rel="noopener noreferrer">
-              Metamask extension.
-            </a>{' '}
-            If you have installed refresh the page.
-          </span>
-        ),
-      });
+      Notifications('Metamasknotinstalled', network, network);
     }
-  }, [arbitrumNetworkConnect]);
-
-  const handleConnect = useCallback(async () => {
-    if (network === Network.Ethereum) {
-      handleEthereumConnection();
-    } else if (network === Network.Arbitrum) {
-      handleArbitrumConnection();
-    }
-  }, [handleArbitrumConnection, handleEthereumConnection, network]);
+  }, [network]);
 
   const handleCancel = useCallback(async () => {
     setSubmitted(false);
-  }, []);
+    if (network === Network.Arbitrum) {
```

Comment:
The top if checks if the user is connected to any Automate network (if (connectedNetwork !== 'none') 
The inner one checks if the user is connected to correct Automate network ( if (connectedNetwork === network.toLowerCase())

Reply:
for example if user is connected to binance's network, the first if will give "You're not connected to Automate.." error notification.
if the user is connected to Automate's Eth. Network the second inner if will give "... Please change it to Automate's... network" error notification.
Actually, I can use one if than two ifs inside but i believe it will take more lines of code than this approach (now its just 9 lines) It will be like, if network is arb else if eth, else if arb.rinkeby else if eth.ropsten else not connected to automate.

Reply:
OK but why in `handleCancel`? This function is just to dismiss the modal basically so this extra logic has no place here